syntax.md

```diff
@@ -0,0 +1,168 @@
+Syntax
+===
+
+# Sequences
+
+## Notes
+
+This plays an `e` note in the default octave (4) with the piano instrument in a loop:
+```
+"e"
+  >> piano
+  end
```

Comment:
Still not sure about the `end` keyword, but it seems like it's the easiest solution to make sure the compiler knows which level of hierarchy we are in.

Reply:
Still think that the even easier solution is to not allow modifiers inside an instrument's play function. This would also fix some of the professor's issues with inconsistency